python/tutorial.py

```diff
@@ -35,6 +35,7 @@
 # SNIPPET_START: anonymous_requests
 r = anonymous_client.get("/node/state")
 assert r.status_code == http.HTTPStatus.OK
+# r.body is HTTP response body
```

Comment:
If we have the right Sphinx, I think the text surround the snippet should instead link to the api doc for the Response type.